Libplanet/Net/Messages/Message.cs

```diff
@@ -77,6 +72,11 @@ internal enum MessageType : byte
             /// Contains the calculated recent states and state references.
             /// </summary>
             RecentStates = 0x0c,
+
+            /// <summary>
+            /// Message containing demand block hashes with their index numbers.
+            /// </summary>
+            BlockHashes = 0x0d,
```

Comment:
As the message coding became incompatible with what it had been, I assigned the new message ID number to this so that two types (old and new) of `BlockHashes` don't be mixed or used at a time on the network.